Latex/metadaten.sty

```diff
@@ -7,35 +7,35 @@
 %Autor 3:
 \newcommand{\autordrei}{Name 3}
 %Autoren - für PDF Titel
-\newcommand{\autoren}{Namen, viele Namen}
+\newcommand{\autoren}{{\autoreins}, {\autorzwei}, \autordrei}
```

Comment:
warum Klammern um um Autor 1 und 2 aber nicht 3? Bzw. warum überhaupt?

Reply:
Das Komma wird sonst unter Umständen zum Variablen-Namen dazu gezählt... Außerdem darf das in Klammern nicht umgebrochen werden. Sieht also schöner aus, weil z.b. die Namen immer auf einer Zeile stehen...
Aber Hauptsächlich wegen Komma zur Variable... Beim dritten daher nicht --> da kein komma :D

Reply:
ich hätte einfach noch n Leerzeichen zwischen command und "," gesetzt, aber so gehts natürlich auch. mit Leerzeichen siehts nur einheitlicher aus. 
EGAAAL